examples/4_GPU/plot_gpu.py

```diff
@@ -6,16 +6,16 @@
 
 Example using GPU calculation with :py:meth:`~radis.lbl.SpectrumFactory.eq_spectrum_gpu`
 
-This method requires a GPU - Currently, only Nvidia GPU's are supported.
-For more information on how to setup your system to run GPU-accelerated methods
-using CUDA, check :ref:`GPU Spectrum Calculation on RADIS <label_radis_gpu>`
-
 .. note::
 
-    in the example below, the code runs on the GPU by default. In case no Nvidia GPU is
-    detected, the code will instead be ran on CPU. This can be toggled manually by setting
-    the ``backend`` keyword either to ``'gpu-cuda'`` or ``'cpu-cuda'``.
-    The run time reported below is for CPU.
+    Systems with a dedicated GPU often have multiple devices available, because they also have an integrated GPU in the
+    main processor. This can be selected by chosing a different device_id.
```

Comment:
perhaps mention that device_id can be integer or str

Reply:
Done